org.eclipse.xtext.tests/src-gen/org/eclipse/xtext/parser/antlr/services/Bug301935TestLanguageGrammarAccess.java

```diff
@@ -38,7 +38,8 @@
 		//	value2=ID WS;
 		@Override public ParserRule getRule() { return rule; }
 		
-		//name=ID WS value=ID NL value2=ID WS
+		//name=ID WS value=ID NL
+		//value2=ID WS
```

Comment:
To me looks more and more like an off-by-one error in the formatter.

Reply:
But this is expected ... newline is preserved ...

Reply:
Roger that